core/new-gui/src/styles.scss

```diff
@@ -40,3 +41,7 @@ blockquote {
 .ant-table-pagination {
   justify-content: center;
 }
+
+.mat-expansion-panel-body {
+  padding: 0 !important;
+}
```

Comment:
This file is the global stype css. is there somewhere local we can keep this overwrite?

Reply:
No. This CSS has to be here to make it work.